ffi/util.lua

```diff
@@ -546,6 +546,10 @@ function util.isAndroid()
     return isAndroid
 end
 
+function util.isPocketbook()
+    return lfs.attributes("/ebrmain")
```

Comment:
Might need something more specific ;).

Bookeen might still have traces of this somewhere, as do potentially old Kobos.

Reply:
Hmm, what about `/ebrmain/pocketbook`? This one is a symlink that points to cramfs image is mounted, ie leftovers are unlikely to have valid one.

Reply:
Yup, that would do the trick ;).

Reply:
Done.